packages/rps/package.json

```diff
@@ -23,7 +22,7 @@
     "eslint:write": "eslint \"**/*.{js,ts,tsx}\" --fix --max-warnings=0",
     "start": "yarn contracts:compile && node scripts/start.js",
     "start:shared-ganache": "NODE_ENV=development npx start-shared-ganache",
-    "prepare": "yarn build:ci",
+    "prepare": "run-s clearContracts build:typescript contracts:compile",
```

Comment:
I don't think `clearContracts` is needed.

Reply:
Good call.